jadx-cli/src/main/java/jadx/cli/JadxCLIArgs.java

```diff
@@ -130,14 +130,18 @@ public boolean processArgs(String[] args) {
 	 * Used to merge saved options and options passed in command line.
 	 */
 	public boolean overrideProvided(String[] args) {
-		JCommanderWrapper<JadxCLIArgs> jcw = new JCommanderWrapper<>(new JadxCLIArgs());
+		JCommanderWrapper<JadxCLIArgs> jcw = new JCommanderWrapper<>(newInstance());
 		if (!jcw.parse(args)) {
 			return false;
 		}
 		jcw.overrideProvided(this);
 		return process(jcw);
 	}
 
+	protected JadxCLIArgs newInstance() {
+		return new JadxCLIArgs();
+	}
```

Comment:
This method not needed.
And please, don't change code not needed for your fix.

Reply:
This change is required to make the parameter `cmdSelectClass` in JadxSettings work (used by  `JadxCLIArgs.overrideProvided(..)` . 

Or do you prefer creating a new instance via reflection or may be `clone()`?